internal/pkg/aws/profile/profile.go

```diff
@@ -47,7 +47,7 @@ func (c *Config) Names() []string {
 	var profiles []string
 	for _, section := range c.f.Sections() {
 		// Named profiles created with "aws configure" are formatted as "[profile test]".
-		profiles = append(profiles, strings.TrimSpace(strings.ReplaceAll(section, "profile", "")))
+		profiles = append(profiles, strings.TrimSpace(strings.TrimPrefix(section, "profile")))
```

Comment:
Should we add an extra space at the end of "profile "?
```suggestion
		profiles = append(profiles, strings.TrimSpace(strings.TrimPrefix(section, "profile ")))
```

Reply:
It'd be amazing if we can add a test to make sure "[profile profile1]"  is going to be returned as "profile1" but no worries otherwise!
We can do a fast-follow up with a test file.